hydromt_fiat/workflows/exposure_vector.py

```diff
@@ -538,52 +548,89 @@ def setup_ground_floor_height(
 
     def setup_max_potential_damage(
         self,
-        max_potential_damage: Union[List[float], float],
-        damage_types: Union[List[str], str, None] = None,
-        country: Union[str, None] = None,
-    ):
-        if max_potential_damage == "jrc_damage_values":
```

Comment:
Please keep the part from line 545-550 and 553-557 in this function, that you removed. They are required when people use hazard or JRC as a source for the max potential damages. Your part should be added on top; so making more options possible.